src/main/java/com/box/sdk/BoxSharedLink.java

```diff
@@ -158,7 +187,7 @@ public void setAccess(Access access) {
      */
     public void setPassword(String password) {
         this.password = password;
-        this.addPendingChange("password", password.toString());
```

Comment:
Another strange bit of code.  Am I missing something here?  Why was .toString() called on an object known to always be a string?

Reply:
I have no idea... This doesn't make any sense. I think it was just a mistake and a warning I want to eliminate.